console/console-init/ui/src/modules/address/AddressPage.tsx

```diff
@@ -17,14 +17,13 @@ import { Divider } from "@patternfly/react-core/dist/js/experimental";
 import { useQuery, useMutation } from "@apollo/react-hooks";
 import { StyleSheet } from "@patternfly/react-styles";
 import { ISortBy } from "@patternfly/react-table";
-import { AddressListToolbar } from "./containers/AddressListToolbar";
 import { AddressListContainer } from "./containers/AddressListContainer";
 import {
   CURRENT_ADDRESS_SPACE_PLAN,
   DELETE_ADDRESS,
   PURGE_ADDRESS
 } from "graphql-module/queries";
-import { IAddress } from "./components/AddressList";
+import { IAddress } from "modules/address/components/AddressList";
```

Comment:
we are avoid relative path ""../../  not current path. so you can use current path ./. pevious import was not wrong you can use that.
import { IAddress } from "./components/AddressList";